src/engine/ruleset/decoders/fortinet/fortimanager.yml

```diff
@@ -0,0 +1,201 @@
+---
+name: decoder/fortinet-fortimanager/0
+
+metadata:
+  title: Decoder for Fortinet FortiManager Endpoint Security
+  description: Decoder for Fortinet FortiManager Endpoint Security
```

Comment:
Improve-Documentation: Please add the module version

Reply:
Done